.pantheon/pantheon.yml

```diff
@@ -0,0 +1,28 @@
+# Put overrides to your pantheon.upstream.yml file here.
+# For more information, see: https://pantheon.io/docs/pantheon-yml/
+api_version: 1
+
+# See https://pantheon.io/docs/pantheon-yml#protected-web-paths for usage.
+protected_web_paths:
+  - /wp-content/.gitignore
+  - /wp-content/.gitmodules
+  - /wp-content/.phpcs.xml
+  - /wp-content/.pantheon
+  - /wp-content/vendor
+  - /wp-content/composer.json
+  - /wp-content/composer.lock
+  - /wp-content/README.md
+
+# See https://pantheon.io/docs/pantheon-yml/#enforce-https--hsts for valid values.
+enforce_https: full
+
+# See https://docs.pantheon.io/pantheon-yml#php-version
+php_version: 8.1
```

Comment:
If we're going to use 8.1 here we ought to use it in this project as a whole. Could you set this to 8.0 for now and file an issue to bump the repo to 8.1? We'll need to update composer.json and rebuild the lockfile.

Reply:
Alternately, we could go directly to 8.2. It's available on Pantheon and recommended.